docs/GroupDefs.hpp

```diff
@@ -48,6 +48,48 @@
 /*!
  * \defgroup ControlSystemGroup Control System
  * \brief Contains control system elements
+ *
+ * The control system manages the time-dependent mapping between frames, such as
+ * the fixed computational frame (grid frame) and the inertial frame. The
+ * time-dependent parameters of the mapping are adjusted by a feedback control
+ * system in order to follow the dynamical evolution of the black holes. In BBH
+ * simulations the map is typically a composition of maps that include
+ * translation, rotation, scaling, shape, etc. Each map under the governance of
+ * the control system has an associated time-dependent map parameter
+ * \f$\lambda(t)\f$ that is regularly updated by a control signal \f$U(t)\f$,
+ * which is determined by an error function \f$Q(t)\f$ that is uniquely
+ * specified for each map. The control system is designed to minimize the error,
+ * \f$Q(t) \rightarrow 0\f$. This is accomplished by updating the Nth derivative
+ * of the piecewise polynomial FunctionOfTime associated with the parameter
+ * \f$\lambda(t)\f$. Note that \f$\lambda(t)\f$, \f$U(t)\f$, and \f$Q(t)\f$ can
```

Comment:
There is no indication of what `U(t)` actually means. Please indicate that the Nth derivative of the FunctionOfTime is set to `U(t)` (or is it minus `U(t)`?), and this is how `U(t)` actually controls the FunctionOfTime.

Also, this is too wordy. How about from line 57 onward:
Each map under the governance of  the control system has an associated time-dependent map parameter lambda(t) that is a piecewise Nth order polynomial.  At regular time intervals the control system resets the Nth time derivative of lambda(t) to a new constant value, in order to minimize an error function Q(t) that is specific to each map.  At each reset time, the Nth derivative of lambda(t) is set to a function U(t), called the control signal, that is determined by Q(t) and its time derivatives. Note that...